collector/src/test/java/io/prometheus/jmx/CamelMBean.java

```diff
@@ -0,0 +1,23 @@
+package io.prometheus.jmx;
+
+import javax.management.MBeanServer;
+import javax.management.ObjectName;
+
+public interface CamelMBean {
+    long getLastExchangeFailureTimestamp();
+}
+
+class Camel implements CamelMBean{
+    public static void registerBean(MBeanServer mbs)
+            throws javax.management.JMException {
+        ObjectName mbeanName = new ObjectName(
+                "org.apache.camel:context=my-camel-context,type=routes,name=\"my-route-name\"");
+        Camel mbean = new Camel();
+        mbs.registerMBean(mbean, mbeanName);
+    }
+
+    @Override
+    public long getLastExchangeFailureTimestamp() {

```

Comment:
This isn't a Date

Reply:
fixed the test :)
thank you very much for the feedbacks and could you review the latest commit please, thank you!